CMakeLists.txt

```diff
@@ -150,6 +150,7 @@ option(COVERAGE "Build with instrumentation to record code coverage." OFF)
 option(BUILD_64 "Build 64 bit version of editor" ON)
 option(BUILD_AUTOUPDATE "Build with autoupdate support" OFF)
 option(BUILD_CRASH_REPORTER "Build with crash reporter" OFF)
+option(BUILD_PORTABLEAPPS "Build for Windows PortableApps.com" OFF)
```

Comment:
Windows in the current location feels strange. How about

"Windows build for PortableApps.com" # my preference
or
"Build for PortableApps.com" # assumes knowledge that PortableApps.com is Windows-only

Reply:
Thank you for your comment.
Unfortunately, English is not my first language so I often make mistakes.
I updated the PR accoding to your comment.
Ciao,
ABL